src/app/configurations/page.tsx

```diff
@@ -220,6 +220,189 @@ const providerModels = {
     "phi3:3.8b",
     "phi3:14b",
   ],
+  openrouter: [
```

Comment:
Will need to refactor this after my PR goes in.

https://github.com/HamedMP/CursorLens/pull/5

Reply:
Also means pricing metrics for these open router models will not be supported

Reply:
Oh, let's fix that, how we can do it?

Reply:
TLDR: option 3.

Option 1: we duplicate the entire object body, for each model configuration, onto the model that is available as a proxy via open router.

- pro: no further refactors from my PR
- con: repeated config - only if the provider model and proxied model identical in price.

Option 2: we instead put an attribute on the object body of models that have availability via proxies 

- pro: avoids repeated config
- con: separation of concerns if the configs do differ in price between provider and proxies model

Option 3: we split the provider / model list config away from the config details and instead reference the config in the original provider / model and repeat it in the proxies (open router).

- pro: can assign the default model config directly to the source providers offering. Can `{ ...spread, override: "specific values" }` like price if they different slightly or keep identical default model config if the same
- con: requires further change to our model config layout (not an issue)

Conclusion: we go with option 3.

Reply:
OpenAI pricing is same too. No additional markup for using it via open router (OR) proxy.

And other models like Nous Hermes don't offer their own hosted service. So those prices are what OR sets.

There are other proxies offering a similar service to OR like  AI/ML API

https://aimlapi.com/models/nous-hermes-2-mistral-dpo-7b?43d56c14_page=2&6c9afc24_page=19

Their prices are higher than OR so I see no point in considering them.